### PR TITLE
Great progress

### PR DESCRIPTION
**Pull request by dzenanz**
Ti36 test has mean error of only 10 or so, due to several completely failed registrations.

### CODE REVIEW

File: include/itkTileMontage.h [hunk 1]
```diff
@@ -170,7 +192,7 @@ class ITK_TEMPLATE_EXPORT TileMontage: public ProcessObject
   using ReaderType = itk::ImageFileReader< ImageType >;
 
   /** Just get image pointer if the image is present, otherwise read it from file. */
-  ImageType* GetImage(TileIndexType nDIndex);
+  ImageType* GetImage(TileIndexType nDIndex, bool metadatOnly);
```
Comment (thewtex) [inline include/itkTileMontage.h]: matadaOnly -> metadataOnly

File: include/itkPhaseCorrelationOperator.h [hunk 2]
```diff
@@ -69,8 +66,31 @@ class ITK_TEMPLATE_EXPORT PhaseCorrelationOperator :
...
+   * are transition zones with linearly increasing (p0-p1)
+   * and decreasing (p2-p3) coefficient of influence. Function plot:
+   *   ^
+   *  1|             _________
+   *   |            /         \
+   *   |           /           \
+   *   |          /             \
+   *  0|    *----*---*-------*---*----*
+   * coef  0.0   p0  p1      p2  p3  1.0
+   */
```
Comment (thewtex) [inline include/itkPhaseCorrelationOperator.h]: Use a Butterworth filter here to prevent phase distortion.

**Comment by dzenanz** [comment]
On my Synthetic3D test passes with MKL back-end:
```text
18: Test command: C:\Misc\BlueQuartz\ITK-vs2017\bin\Release\MontageTestDriver.exe "itkMontagePCMTestFiles" "C:/Misc/BlueQuartz/ITKMontage-vs2017-ITKmaster/ExternalData/test/Input/Synthetic/Cuboid1.mha" "C:/Misc/BlueQuartz/ITKMontage-vs2017-ITKmaster/ExternalData/test/Input/Synthetic/Cuboid2.mha" "C:/Misc/BlueQuartz/ITKMontage-vs2017-ITKmaster/test/../Testing/Temporary/itkMontagePCMTestFilesSynthetic3D.nrrd" "0.0" "0.0" "0.0" "-21.0" "35.0" "1.0"
18: Test timeout computed to be: 1500
18: Padding method 0
18: -21.003 == -21 == -21.003
18: 34.9999 == 35 == 34.9999
18: 1 == 1 == 1
18: Padding method 1
18: -21.003 == -21 == -21.003
18: 34.9999 == 35 == 34.9999
18: 1 == 1 == 1
18: Padding method 2
18: -21.003 == -21 == -21.003
18: 34.9999 == 35 == 34.9999
18: 1 == 1 == 1
18:
18: Test PASSED.
1/1 Test #18: itkMontagePCMTestFilesSynthetic3D ...   Passed    0.27 sec
```

With FFTW this fails. With VNL FFT back-end some additional tests fail.

Additional commits fix the synthetic 3D test failure.

File: include/itkPhaseCorrelationOperator.h (hunk 2 above)
Comment (dzenanz) [reply include/itkPhaseCorrelationOperator.h]: This is still planned to be done, just not in this PR.

**Pull request merged**